receiver/filestatsreceiver/Makefile

```diff
@@ -1 +1,38 @@
 include ../../Makefile.Common
+
+WEAVER_CONTAINER=otel/weaver@sha256:0466739a9c9bba8273ef5767ed2b417b82ef652d4e5fd3498185c714fa809460
```

Comment:
How can we keep this up-to-date with new releases of weaver?

Reply:
And an idea (feel free to ignore), could we use the Rust crate instead of a container? I'm always having problems with docker in my mac 😭

Reply:
I skipped the intense thing that Josh had done for his talk, where he had a docker setup that would be picked up by renovate for upgrades. Here it is: https://github.com/jsuereth/o11y-by-design/blob/main/Makefile#L2-L8

I don't mind crates and stuff, and github actions, and whatnot. We need to have an issue about it and keep it going.

Reply:
https://github.com/open-telemetry/weaver/issues/879